boltdb/boltdb_manager.go

```diff
@@ -0,0 +1,95 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package boltdb
+
+import (
+	"context"
+	"fmt"
+	"github.com/google/ts-bridge/storage"
+	"os"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/timshannon/bolthold"
+)
+
+// Options holds settings specific to boltdb
+type Options struct {
+	DBPath string
+}
+
+// Manager struct implementing the storage.Manager interface
+type Manager struct {
+	Store *bolthold.Store
+}
+
+// New initializes the Manager struct implementing a generic storage.Manager interface
+func New(_ context.Context, options *Options) *Manager {
+	// If path for a DB is not configured, default to working directory.
+	var boltPath string
```

Comment:
I'd suggest just doing `boltPath := options.DBPath` here, and using boltPath throughout the function.

Note that currently you seem to be ignoring `options.DBPath` it it's set (and there is no test catching this).

Reply:
Yeah, that's my fault for not creating a test. Fixed and created a test for it.